### PR TITLE
add animal demos in readme

### PR DESCRIPTION
**Pull request by jin-s13**
1. Support selecting category for mmdet models.
2. Support video demos with full-image input.
3. Add tutorials for generating animal demos.
4. Add demos in readme.

### CODE REVIEW

File: demo/top_down_full_img_video_demo.py [hunk 1]
```diff
@@ -0,0 +1,111 @@
+import os
```
Comment (innerlee) [inline demo/top_down_full_img_video_demo.py:1]: What does the "full" in filename mean?

File: demo/top_down_img_demo_with_mmdet.py [hunk 2]
```diff
@@ -66,6 +66,11 @@ def main():
         help='Bounding bbox score threshold')
     parser.add_argument(
         '--kpt-thr', type=float, default=0.3, help='Keypoint score threshold')
+    parser.add_argument(
+        '--cat-id',
```
Comment (innerlee) [inline demo/top_down_img_demo_with_mmdet.py:70]: det-cat-id

**Review by innerlee — APPROVED** [review]

File: README.md [hunk 3]
```diff
@@ -34,7 +34,8 @@ The master branch works with **PyTorch 1.3+**.
 
 - **Support diverse tasks**
 
-  We support a wide spectrum of mainstream human pose analysis tasks in current research community, including 2d multi-person human pose estimation, 2d hand pose estimation, 2d face landmark detection, 133 keypoint whole-body human pose estimation, fashion landmark detection and 3d human mesh recovery.
+  We support a wide spectrum of mainstream pose analysis tasks in current research community, including 2d multi-person human pose estimation, 2d hand pose estimation, 2d face landmark detection, 133 keypoint whole-body human pose estimation, 3d human mesh recovery, fashion landmark detection and animal pose estimation.
```
Comment (innerlee) [inline README.md:37]: Adding animal demo to this page, too

File: demo/top_down_full_img_video_demo.py (hunk 1 above)
Comment (jin-s13) [reply demo/top_down_full_img_video_demo.py:1]: No object detection model used. 
Directly input the full size image (without bbox cropping) into the model.

Comment (innerlee) [reply demo/top_down_full_img_video_demo.py:1]: "img_video" means img or video, or means "full_img" version of "video"

Comment (jin-s13) [reply demo/top_down_full_img_video_demo.py:1]: Aha yes, "full_img" version of "video".

Comment (innerlee) [reply demo/top_down_full_img_video_demo.py:1]: How about "top_down_video_demo_full_frame_without_det"

**Pull request merged**